### ISSUE TITLE
paranoid attribute is not being set to true in models

### ISSUE THREAD

**Issue by aeharake** [sequelize/sequelize-auto#544]
I have this options.json file:

```
{
  "timestamps": false,
  "deletedAt": "deleted_at",
  "createdAt": "created_at",
  "updatedAt": "updated_at",
  "paranoid": true
}

```

After running the following command:

`sequelize-auto -h 127.0.0.1 -d dbname -u username -x password -p 3306  --dialect mysql -o "./src/database/models" -l ts -a "./src/options.json"`

I should be able to see that paranoid is turned on, however, this is not happening.

**Comment by steveschmitt** [comment]
Duplicate of #529

**Comment by aeharake** [comment]
> Duplicate of #529

Is anybody working on a fix?

**Comment by steveschmitt** [comment]
Fixed in 0.8.4
